api/envoy/config/cluster/v3/cluster.proto

```diff
@@ -412,8 +412,8 @@ message Cluster {
     // The table size for Maglev hashing. The Maglev aims for ‘minimal disruption’ rather than an absolute guarantee.
     // Minimal disruption means that when the set of upstreams changes, a connection will likely be sent to the same
     // upstream as it was before. Increasing the table size reduces the amount of disruption.
-    // The table size must be prime number. If it is not specified, the default is 65537.
-    google.protobuf.UInt64Value table_size = 1;
+    // The table size must be prime number limited to 4M. If it is not specified, the default is 65537.
```

Comment:
5M? the constraints below doesn't match comment here.

Reply:
just a random prime number larger than 4M, nothing special here.

Reply:
Since we publish this documentation separately from the proto source, it should contain the actual, exact limit.

Reply:
Thanks, fixed.